scripts/provisioning/s3_setup

```diff
@@ -330,18 +272,20 @@ if [[ $config_ldap == true ]]
 then
     /opt/seagate/cortx/s3/install/ldap/setup_ldap.sh --defaultpasswd --skipssl
 
-    # read host_names from confstore
-    # it will be a comma-separated list of FQDN
-    cluster_hosts=$(s3confstore --getkey "cluster>cluster_hosts" --path "json:///opt/seagate/cortx/s3/conf/s3_confstore.json")
-
-    IFS=','
-    read -ra hosts_arr <<< "$cluster_hosts"
-    if [ ${#hosts_arr[@]} -gt 1 ]
+    # check number of nodes in the cluster
+    server_nodes_count=$(s3confstore --getkey "cluster>node_count" --path "$confstore_config_url")
+    if [ "$server_nodes_count" -gt 1 ]
     then
+        for ((index=0; index<"$server_nodes_count"; index++))
+        do
+            host=$(s3confstore --getkey "cluster>server["$index"]>hostname" --path "$confstore_config_url")
```

Comment:
can we catch error if s3confstore fails

Reply:
will add check if s3onfstore failed or not.